api/ServiceWorkerGlobalScope.json

```diff
@@ -997,10 +997,10 @@
               "version_added": "81"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "65"
```

Comment:
This is suspect, https://developer.mozilla.org/en-US/docs/Web/API/Window/messageerror_event#browser_compatibility is much earlier. Can you investigate?

Reply:
I did a quick search and found the corresponding bug for this, which confirms Firefox 65: https://bugzilla.mozilla.org/show_bug.cgi?id=1399446

Reply:
I see, so it really was implemented separately and later. I checked https://hg.mozilla.org/mozilla-central/rev/988f40d0fd2c and it also includes actually firing the event, it wasn't just the `onmessageerror` property being added.